f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -149,7 +149,12 @@
         'esd_auto_refresh',
         default=True,
         help='Enable ESD file periodic refresh'
-    )
+    ),
+    cfg.StrOpt(
+        'f5_http_profile_name',
```

Comment:
This file will not only include http_profile configuration, but also other profiles and other resources in the future. We may use a generic name, such as 'f5_extended_definition' or others.

Reply:
We may also ship this json file in agent package, with some default values.

Reply:
> We may also ship this json file in agent package, with some default values.

yes.

Reply:
> such

yes.

Reply:
> This file will not only include http_profile configuration, but also other profiles and other resources in the future. We may use a generic name, such as 'f5_extended_definition' or others.

yes